Cargo.toml

```diff
@@ -16,18 +16,28 @@ categories = ["encoding"]
 [workspace]
 members = ["derive"]
 
+[features]
+default = []
+chrono = ["dep:chrono"]
+geo = ["dep:geo"]
+ordered-float = ["dep:ordered-float"]
+regex = ["dep:regex"]
+roaring = ["dep:roaring"]
+rust_decimal = ["dep:rust_decimal"]
+uuid = ["dep:uuid"]
```

Comment:
I don't think it's necessary to redefine the feature flags if we are reusing the crate name as-is. These are already implied by `optional = true`.

Reply:
Ah ok cool. How would a user specify that they want the `uuid` dependency feature enabled in their Cargo.toml file?

Reply:
Ok, just removed these explicit features ✅